patchdeps.py

```diff
@@ -118,20 +118,17 @@ def get_changesets(args):
             for line in lines:
                 yield GitRev(*line.split(' ', 1))
 
-def print_depends(patches, depends):
+def print_depends(patches: list[Changeset], depends: dict[Changeset, dict[Changeset, Depend]]) -> None:
     for p in patches:
         if not depends[p]:
             continue
-        print("%s depends on: " % p)
+        print(f"{p} depends on: ")
         for dep in patches:
             if dependency := dependencies.get(dep):
                 desc = dependency.desc
-                if desc:
-                    print("  %s (%s)" % (dep, desc))
-                else:
-                    print("  %s" % dep)
+                print(f"  {dep}{f' ({desc})' if desc else ''}")
```

Comment:
In commit " refactor: Convert to f-string ", these nested f-strings are very hard to read IMHO (took me a while to figure out what was going on here), I prefer the original code with if else (which is more verbose than is nice, but is easier to read).